server/apps/distributed-app/docker-compose.yml

```diff
@@ -37,47 +32,37 @@ services:
       - discovery.type=single-node
       - DISABLE_INSTALL_DEMO_CONFIG=true
       - DISABLE_SECURITY_PLUGIN=true
-    networks:
-      james:
-        aliases:
-          - elasticsearch
 
   cassandra:
-    image: cassandra:4.1.3
+    image: cassandra:3.11.10
     ports:
       - "9042:9042"
-    healthcheck:
-      test: [ "CMD", "cqlsh", "-e", "describe keyspaces" ]
-      interval: 3s
-      timeout: 20s
-      retries: 5
-    networks:
-      - james
 
   tika:
-    image: apache/tika:2.8.0.0
-    networks:
-      - james
+    image: apache/tika:1.28.2
 
   rabbitmq:
-    image: rabbitmq:3.12.1-management
+    image: rabbitmq:3.9.18-management
     ports:
       - "5672:5672"
       - "15672:15672"
-    networks:
-      - james
+
+  pulsar:
+    image: apachepulsar/pulsar:2.10.1
+    container_name: pulsar
```

Comment:
Let's create another docker-compose specific for Pulsar MailQueue (e.g `docker-compose-with-pulsar.yml`) and do not change the current `docker-compose.yml`.

Rationale: not everyone uses Pulsar.